extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -104,7 +104,6 @@ tags: {  # <amp-story>
     child_tag_name_oneof: "AMP-GEO"
     child_tag_name_oneof: "AMP-PIXEL"
     child_tag_name_oneof: "AMP-SIDEBAR"
-    child_tag_name_oneof: "AMP-STORY-ACCESS"
```

Comment:
Blocking for I2D/I2R.

Reply:
Unblocking. This does not affect any publishers.

For Googlers, if you're interested in running such a test please see http://go/amp-rule-change